core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-uve-toolbar/components/dot-ema-bookmarks/dot-ema-bookmarks.component.html

```diff
@@ -1,12 +1,16 @@
 @if (store.$previewMode()) {
     <p-button
+        pTooltip="Bookmark"
```

Comment:
Is it necessary to translate this tooltip?

Reply:
I have PR covering this https://github.com/dotCMS/core/pull/31067/